debian/control

```diff
@@ -8,13 +8,13 @@ Homepage: https://truenas.com
 
 Package: python3-google-api-core
 Architecture: amd64
-Depends: python3-googleapis-common-protos (<< 2.0dev), python3-googleapis-common-protos (>= 1.6.0), python3-protobuf (>= 3.12.0), python3-google-auth (<< 2.0dev), python3-google-auth (>= 1.21.1), python3-requests (<< 3.0.0dev), python3-requests (>= 2.18.0), python3-setuptools (>= 34.0.0), python3-six (>= 1.13.0), python3-tz, python3-cachetools (<< 5.0), python3-cachetools (>= 2.0.0), python3-pyasn1-modules (>= 0.2.1), python3-rsa (<< 5), python3-rsa (>= 3.1.4), python3-chardet (<< 4), python3-chardet (>= 3.0.2), python3-idna (<< 3), python3-idna (>= 2.5), python3-urllib3 (<< 1.26), python3-urllib3 (>= 1.21.1), python3-certifi (>= 2017.4.17), python3-pyasn1 (<< 0.5.0), python3-pyasn1 (>= 0.4.6)
+Depends: python3-google-auth (<< 2.0dev), python3-google-auth (>= 1.21.1), python3-googleapis-common-protos (<< 2.0dev), python3-googleapis-common-protos (>= 1.6.0), python3-protobuf (>= 3.12.0), python3-tz, python3-requests (<< 3.0.0dev), python3-requests (>= 2.18.0), python3-setuptools (>= 40.3.0), python3-six (>= 1.13.0), python3-cachetools (<< 5.0), python3-cachetools (>= 2.0.0), python3-pyasn1-modules (>= 0.2.1), python3-rsa (<< 5), python3-rsa (>= 3.1.4), python3-certifi (>= 2017.4.17), python3-chardet (<< 5), python3-chardet (>= 3.0.2), python3-idna (<< 3), python3-idna (>= 2.5), python3-urllib3 (<< 1.27), python3-urllib3 (>= 1.21.1), python3-pyasn1 (<< 0.5.0), python3-pyasn1 (>= 0.4.6)
```

Comment:
I did not foresee that we'll be getting unreadable diffs. Can we also please have a stable item order for these lists?

Reply:
@themylogin i am sorting the dependencies now, so moving on for future diffs we can ensure we will have a defined order, please let me know if you would like me to read the existing file otherwise and do it. Thank you